app/src/main/java/org/fossasia/pslab/activity/AxisDetectionHelpActivity.java

```diff
@@ -0,0 +1,16 @@
+package org.fossasia.pslab.activity;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+import org.fossasia.pslab.R;
+
+public class AxisDetectionHelpActivity extends AppCompatActivity {
```

Comment:
Can we have this as a bottomsheet guide like other instruments have?

Reply:
Sure.